startup_scripts/000_users.py

```diff
@@ -13,6 +13,12 @@
         user = User.objects.create_user(
             username=username,
             password=user_details.get("password", 0) or User.objects.make_random_password(),
+            first_name=user_details.get("first_name", "no-name"),
+            last_name=user_details.get("last_name", "no-lastname"),
+            email=user_details.get("email", "none"),
```

Comment:
The default for `first_name`, `last_name` and `email` should probably be empty string "" as they are allowed to be blank.

Reply:
fair enough, I'll fix it